config/helm/aws-node-termination-handler/templates/daemonset.linux.yaml

```diff
@@ -43,7 +43,7 @@ spec:
           hostPath:
             path: {{ .Values.procUptimeFile | default "/proc/uptime" | quote }}
         {{- if and .Values.webhookTemplateConfigMapName .Values.webhookTemplateConfigMapKey }}
-        - name: "webhookTemplate"
+        - name: "webhook-template"
```

Comment:
looks like these got reverted to what they were before.

Reply:
I think it's not, this PR is intended to change those camel cases into kebab cases. Please refer the original diff in eks-charts PR.

Reply:
ah oops, you're right. Sorry for the delay on this! Thanks!!